hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/token/impl/AccountStore.java

```diff
@@ -78,7 +80,28 @@ public KeyOrLookupFailureReason getKey(final AccountID idOrAlias) {
         if (account.isEmpty()) {
             return new KeyOrLookupFailureReason(null, INVALID_ACCOUNT_ID);
         }
-        return new KeyOrLookupFailureReason(account.get().getAccountKey(), null);
+        return validateKey(account.get().getAccountKey());
+    }
+
+    /**
+     * Fetches the account's key from given {@link MerkleAccount} if the {@code receiverSigRequired}
```

Comment:
The javadoc should be updated to reflect the new API here. There is no MerkleAccount or receiverSigRequired in the function signature

Reply:
Fixed Javadocs. For `receiverSigRequired` , I meant if the the flag is set on the account we lookup. Thanks !